doc/user/content/sql/system-catalog.md

```diff
@@ -270,6 +270,7 @@ Field           | Type       | Meaning
 `ls_offset`     | [`bigint`] | The partition's last stable offset on the broker.
 `app_offset`    | [`bigint`] | The offset of the last message passed to materialized + 1.
 `consumer_lag`  | [`bigint`] | Difference between the `hi_offset` and `app_offset`.
+`initial_hi_offset`  | [`bigint`] | The first known partition's high watermark offset on the broker.
```

Comment:
Hah, the "lo" / "hi" / "slo" / "fas" abbreviation thing is a bit of a pet peeve of mine: can we name it `initial_high_offset`?

Reply:
Sure, we can do that. I named it in a way that aligned with the existing names, which come from librdkafka (not sure if they are also part of Kafka.)